tests/test_photoCalib.py

```diff
@@ -594,11 +594,17 @@ def testCalibrateImageConstant(self):
         photoCalib = lsst.afw.image.PhotoCalib(self.calibration, self.calibrationErr)
         result = photoCalib.calibrateImage(maskedImage)
         self.assertMaskedImagesAlmostEqual(expect, result)
+        uncalibrated = photoCalib.uncalibrateImage(result)
+        # High rtol because our specific choice of calib values results in a
+        # very-close-to-but-not-1 value, which results in poor round-tripping.
+        self.assertMaskedImagesAlmostEqual(maskedImage, uncalibrated, rtol=4e-2)
```

Comment:
I'm hoping those parentheses in the calculation were important and you can shrink this after you add them, because that is indeed an extremely high tolerance.